addons/l10n_dk/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,152 @@
+        <field name="parent_id" ref="l10n_dk.account_tax_report"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_a_section_products" model="account.tax.report.line">
+        <field name="name">Rubrik A = varer - Værdien uden moms af varekøb i andre EU-lande (EU-erhvervelser)</field>
+        <field name="tag_name">Rubrik A = varer - Værdien uden moms af varekøb i andre EU-lande (EU-erhvervelser)</field>
+        <field name="sequence" eval="1"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_additional_info"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_a_section_services" model="account.tax.report.line">
+        <field name="name">Rubrik A = ydelser - Værdien uden moms af ydelseskøb i andre EU-lande</field>
+        <field name="tag_name">Rubrik A = ydelser - Værdien uden moms af ydelseskøb i andre EU-lande</field>
+        <field name="sequence" eval="2"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_additional_info"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_b_section_product_eu" model="account.tax.report.line">
+        <field name="name">Rubrik B = varer (Værdien af varesalg uden moms til andre EU-lande. Indberettes til systemet EU-salg uden moms</field>
+        <field name="tag_name">Rubrik B = varer (Værdien af varesalg uden moms til andre EU-lande. Indberettes til systemet EU-salg uden moms</field>
+        <field name="sequence" eval="3"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_additional_info"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_b_section_products_non_eu" model="account.tax.report.line">
+        <field name="name">Rubrik B = varer - Indberettes ikke til EU-salg uden moms. Værdien af fx. installation og montage, fjernsalg og nye transportmidler til ikke-momsregistrerede kunder uden moms til andre EU-lande</field>
+        <field name="tag_name">Rubrik B = varer - Indberettes ikke til EU-salg uden moms. Værdien af fx. installation og montage, fjernsalg og nye transportmidler til ikke-momsregistrerede kunder uden moms til andre EU-lande</field>
+        <field name="sequence" eval="4"/>
+        <field name="parent_id" ref="l10n_dk.account_tax_report_line_additional_info"/>
+        <field name="country_id" ref="base.dk"/>
+    </record>
+    <record id="account_tax_report_line_vat_statement_b_section_services" model="account.tax.report.line">
+        <field name="name">Rubrik B = ydelser - Værdien af visse ydelsessalg uden moms til andre EU-lande. Indberettes til EU-salg uden moms</field>
+        <field name="tag_name">Rubrik B = ydelser - Værdien af visse ydelsessalg uden moms til andre EU-lande. Indberettes til EU-salg uden moms</field>
```

Comment:
should not there be a formula behind this then?  (are there taxes immediately linked to this line?)  (or maybe the = is not what it seems)

Reply:
Danish Tax reporting is a bit strange some times. So the names for the lines in the report a direct copies of the corresponding  sections in the tax reporting system with the Danish Tax authority. I could see that this was also done on some of the other localizations